webcompat/static/css/src/issue-wizard.css

```diff
@@ -308,7 +308,7 @@
   animation-name: slideupandheight;
   max-height: 0;
   opacity: 0;
-  overflow: hidden;
+  overflow: visible;
```

Comment:
Setting `overflow` to `visible` is the same as just removing the `overflow` property (since `visible` is the default).

Can we just remove it?

cc @ksy36 in case she has any idea why that might be a bad idea. In my testing, it didn't matter (at various viewport sizes).

Reply:
Seems to work good, except this button is sticking out now:

<img width="362" alt="Screen Shot 2020-07-21 at 6 39 58 PM" src="https://user-images.githubusercontent.com/1303908/88114691-72c63200-cb82-11ea-9a81-780002b05458.png">

Which is actually https://github.com/webcompat/webcompat.com/issues/3029

We can change `max-height` to be 340px in `slidedownusername` keyframe (instead of 300) and that will fix it @soniasingla  :)